deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr_ocp.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: csi.ibm.com/v1
+kind: IBMBlockCSI
+metadata:
+  name: ibm-block-csi
+  namespace: kube-system
+  labels:
+    app.kubernetes.io/name: ibm-block-csi-operator
+    app.kubernetes.io/instance: ibm-block-csi-operator
+    app.kubernetes.io/managed-by: ibm-block-csi-operator
+spec:
+  # controller is a statefulSet with ibm-block-csi-driver-controller
+  # container and csi-provisioner, csi-attacher and livenessprobe sidecars.
+  controller:
+    repository: ibmcom/ibm-block-csi-driver-controller
+    tag: "1.2.0"
+    imagePullPolicy: IfNotPresent
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/arch
+                  operator: In
+                  values:
+                    - amd64
+                    - s390x
+                    - ppc64le
+
+  # node is a daemonSet with ibm-block-csi-driver-node container
+  # and csi-node-driver-registrar and livenessprobe sidecars.
+  node:
+    repository: ibmcom/ibm-block-csi-driver-node
+    tag: "1.2.0"
+    imagePullPolicy: IfNotPresent
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/arch
+                  operator: In
+                  values:
+                    - amd64
+                    - s390x
+                    - ppc64le
+
+  sidecars:
+  - name: csi-node-driver-registrar
+    repository: registry.redhat.io/openshift4/ose-csi-driver-registrar
+    tag: v4.3
+    imagePullPolicy: IfNotPresent
+  - name: csi-provisioner
+    repository: registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7
```

Comment:
as I understand from the file name the yaml is for installation on ocp, its not a problem that the repository used here is named with rhel7?

Reply:
It's not that I agree with their naming, but I couldn't find a version without it
https://github.com/IBM/ibm-block-csi-operator/blob/7b095f15c347c4dad2c8eaf7b0b2f62ee042ce28/deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.2.0.clusterserviceversion.yaml#L83

Reply:
ok